### PR TITLE
🚀 Andromeda

### PR DESCRIPTION
**Pull request by sunnyvempati**

### CODE REVIEW

File: tomls/omnibus-base-goerli-andromeda/perps/global.toml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+include = ["./feeCollector.toml", "./referrers.toml"]
```
Comment (noisekit) [inline tomls/omnibus-base-goerli-andromeda/perps/global.toml:1]: Suggestion: move includes to the top level andromeda toml

File: tomls/omnibus-base-goerli-andromeda/spot/usdc.toml [hunk 2]
```diff
@@ -5,9 +5,7 @@ defaultValue = "<%= parseEther(String(5_000_000)) %>"
 defaultValue = "60"
 
 [provision.usdc_mock_collateral]
-source = "mintable-token"
-targetPreset = "<%= settings.target_preset %>"
-options.salt = "<%= settings.salt %>"
+source = "mintable-token:1.3"
```
Comment (noisekit) [inline tomls/omnibus-base-goerli-andromeda/spot/usdc.toml:8]: Need to use 1.7 and decimals 6
```suggestion
source = "mintable-token:1.7"
options.decimals = "6"
```

File: tomls/oracles/pyth-snx.toml [hunk 3]
```diff
@@ -7,7 +7,7 @@ target = ["system.oracle_manager.Proxy"]
 func = "registerNode"
 args = [
     5, # 5 = pyth aggregator type
-    "<%= defaultAbiCoder.encode(['address', 'bytes32', 'bool'], [imports.pyth_erc7412_wrapper.contracts.PythERC7412Wrapper.address, settings.pythSnxFeedId, false]) %>",
+    "<%= defaultAbiCoder.encode(['address', 'bytes32', 'bool'], ['0xEbe57e8045F2F230872523bbff7374986E45C486', settings.pythSnxFeedId, false]) %>",
```
Comment (noisekit) [inline tomls/oracles/pyth-snx.toml:10]: need to revert this change
```suggestion
    "<%= defaultAbiCoder.encode(['address', 'bytes32', 'bool'], [imports.pyth_erc7412_wrapper.contracts.PythERC7412Wrapper.address, settings.pythSnxFeedId, false]) %>",
```

File: tomls/omnibus-base-goerli-andromeda/spot/usdc.toml (hunk 2 above)
Comment (noahlitvin) [reply tomls/omnibus-base-goerli-andromeda/spot/usdc.toml:8]: This was deployed as is though? I don't think we should change it if that's the case.

File: tomls/oracles/pyth-snx.toml (hunk 3 above)
Comment (noahlitvin) [reply tomls/oracles/pyth-snx.toml:10]: We need to do more troubleshooting on why it wasn't working per your suggestion before.

File: tomls/omnibus-base-goerli-andromeda/spot/usdc.toml (hunk 2 above)
Comment (noisekit) [reply tomls/omnibus-base-goerli-andromeda/spot/usdc.toml:8]: We need to have fUSDC to have same decimals as real usdc because all the tests and potentially integrators implementations assume 18 decimals. Which will be a problem when we deploy to mainnet. We need to have a testnet deploy that is close to mainnet to avoid such issues

File: tomls/oracles/pyth-snx.toml (hunk 3 above)
Comment (noisekit) [reply tomls/oracles/pyth-snx.toml]: I was present at the deployment. This was a "try" but fid not fix issue. And was not reverted.

File: tomls/omnibus-base-mainnet-andromeda/perps/eth.toml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+defaultValue = "0.5"
+
+[setting.perpsEthSynthMaxCollateralAmount]
+defaultValue = "<%= MaxUint256 %>"
+
+[setting.perpsEthMaxLiquidationPd]
+defaultValue = "0.0005"
+
+[setting.perpsEthEndorsedLiquidator]
+defaultValue = "0x11233749514Ab8d00C0A5873DF7428b3db70030f"
```
Comment (noisekit) [inline tomls/omnibus-base-mainnet-andromeda/perps/eth.toml:55]: This value is the same as testnet. If it should be - nevermind

File: tomls/omnibus-base-mainnet-andromeda/perps/btc.toml [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+defaultValue = "0.5"
+
+[setting.perpsBtcSynthMaxCollateralAmount]
+defaultValue = "<%= MaxUint256 %>"
+
+[setting.perpsBtcMaxLiquidationPd]
+defaultValue = "0.0005"
+
+[setting.perpsBtcEndorsedLiquidator]
+defaultValue = "0x11233749514Ab8d00C0A5873DF7428b3db70030f"
```
Comment (noisekit) [inline tomls/omnibus-base-mainnet-andromeda/perps/btc.toml:54]: Same comment here too: this value is the same as testnet. If it should be - nevermind then

**Review by noisekit — APPROVED** [review]
I did check everything and made a few fixes. There are couple of copypasted addresses to double-check and we still have a few TODOs.
Existing tests are green (NOTE: we don't have tests for mainnet deployment yet!!!)
